app/models/manageiq/providers/base_manager/metrics_capture.rb

```diff
@@ -6,4 +6,152 @@ def initialize(target, ems = nil)
     @target = target
     @ems = ems
   end
+
+  def zone
+    ems.zone
+  end
+
+  def my_zone
+    ems.zone.name
+  end
+
+  # Capture all metrics for an ems
+  def perf_capture
+    perf_capture_health_check
+    targets = Metric::Targets.capture_ems_targets(ems)
+
+    targets_by_rollup_parent = calc_targets_by_rollup_parent(targets)
+    target_options = calc_target_options(targets_by_rollup_parent)
+    targets = filter_perf_capture_now(targets, target_options)
+    queue_captures(targets, target_options)
+
+    # Purge tasks older than 4 hours
+    MiqTask.delete_older(4.hours.ago.utc, "name LIKE 'Performance rollup for %'")
```

Comment:
This looks a little out of place here, maybe we should be cleaning up old ones [where we create the new one](https://github.com/ManageIQ/manageiq/pull/19541/files#diff-62cd5f478c94c6f643d7a8f6d8262251R124)?